docs/spec-files/message_structure.md

```diff
@@ -102,6 +102,10 @@ Headers in [DIDComm Messaging](.) are intended to be extensible in much the same
 
  When Alice sends the same message to Bob and Carol, it is by inspecting this header that Bob and Carol learn that the message was sent to both of them. If the header is omitted, each recipient can only assume they are the only recipient (much like an email sent only to `BCC:` addresses).
 
+For signed messages, notice that it is RECOMMENDED to have a properly defined `to` header, since it prevents certain kind of [forwarding attacks](https://theworld.com/~dtd/sign_encrypt/sign_encrypt7.html), where a message that wasn't meant for a given recipient is forwarded along with its signature to that recipient which then could blindly trust it because of the signature.
```

Comment:
```suggestion
For signed messages, there's specific requirements around properly defined `to` header outlined in the `to` header definition above. The reason for this is to prevents certain kind of [forwarding attacks](https://theworld.com/~dtd/sign_encrypt/sign_encrypt7.html), where a message that wasn't meant for a given recipient is forwarded along with its signature to that recipient which then could blindly trust it because of the signature.
```

Reply:
Changing this so that someone reading this section knows to reference the other section for requirements so we're not redundantly stating normative statements.